lib/routes/landiannews/category.ts

```diff
@@ -0,0 +1,48 @@
+import { Data, DataItem, Route, ViewType } from '@/types';
+import { fetchNewsItems, fetchCategory } from './utils';
+
+export const handler = async (ctx): Promise<Data> => {
+    const slug = ctx.req.param('slug');
+
+    const { id, name } = await fetchCategory(slug);
+
+    const rootUrl = 'https://www.landiannews.com/';
+    const postApiUrl = `${rootUrl}wp-json/wp/v2/posts?_embed&categories=${id}`;
+
+    const items: DataItem[] = await fetchNewsItems(postApiUrl);
+
+    return {
+        title: `${name} - 蓝点网`,
+        description: '给你感兴趣的内容!',
+        link: String(rootUrl),
```

Comment:
`rootUrl` 
https://github.com/DIYgod/RSSHub/blob/ff54862e2a5143044edb6f81787079cd51112220/lib/routes/landiannews/category.ts#L9 is already a string. No need to cast it as a string again.